Add dpkg-buildflags to where configure is really called for effect

Index: eperl/Makefile.PL
===================================================================
--- eperl.orig/Makefile.PL	2011-12-09 01:31:52.000000000 +0100
+++ eperl/Makefile.PL	2011-12-09 01:33:34.000000000 +0100
@@ -29,8 +29,8 @@
 all:
 	\@if [ ! -f Makefile.stand ]; then \\
 	    cp Makefile Makefile.perl; \\
-		echo "${ccarg}./configure --with-perl=$perl"; \\
-	    ${ccarg}./configure --with-perl=$perl; \\
+		echo "${ccarg}./configure \$(shell dpkg-buildflags --export=configure) --with-perl=$perl"; \\
+	    ${ccarg}./configure \$(shell dpkg-buildflags --export=configure) --with-perl=$perl; \\
 	    mv Makefile Makefile.stand; \\
 	    cp Makefile.perl Makefile; \\
 		rm Makefile.perl; \\
